examples/06_form-demo/src/components/funcs.ts

```diff
@@ -0,0 +1,17 @@
+'use server';
+
+import { unstable_rerenderRoute } from 'waku/router/server';
+
+// module state on server
+let message = '';
```

Comment:
Maybe as a "demo", we may want to write it a file, instead of a module variable?

Reply:
good suggestion! I added a file with some text. Let me know what you think of it :)